### PR TITLE
Fix request headers before forwarding

### PR DESCRIPTION
**Pull request by ableuler**
Remove a header from the request before forwarding which can cause problems related to ssl.

### CODE REVIEW

File: test.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"obj(ect)?/[\\d-]+/": {"endpoint": "http://localhost/api", "processor": "app.tests.test_config.DummyProcessor"}}
```
Comment (ableuler) [inline test.json]: Hmm, why is this necessary? Did the tests not work before?

Comment (ciyer) [reply test.json]: I thought that file came from you, but I now see that it came from my commit. I've removed it.
